zoos/zoos-router.js

```diff
@@ -0,0 +1,203 @@
+const router = require("express").Router();
+const knex = require("knex");
+
+const knexConfig = {
+  client: "sqlite3",
+  useNullAsDefault: true,
+  connection: {
+    filename: "./data/lambda.sqlite3"
+  }
+};
+
+const db = knex(knexConfig);
+// endpoints here
+
+// ********** READ METHODS ********** //
+
+// GET zoos root
+
+router.get("/zoos", (req, res) => {
+  db("zoos")
+    .then(zoos => {
+      res.status(200).json(zoos);
+    })
+    .catch(error => {
+      res.status(500).json(error);
+    });
+});
+
+// GET bears root
+
+router.get("/bears", (req, res) => {
+  db("bears")
+    .then(bears => {
+      res.status(200).json(bears);
+    })
+    .catch(error => {
+      res.status(500).json(error);
+    });
+});
+
+// GET zoo by ID
+
+router.get("/zoos/:id", (req, res) => {
+  db("zoos")
+    .where({ id: req.params.id })
+    .then(zoos => {
```

Comment:
Check to make sure there is a zoo that has that id before you send the response.